packages/nextjs/package.json

```diff
@@ -28,9 +28,23 @@
     "ssr.js",
     "app-beta"
   ],
+  "imports": {
+    "#components": {
+      "react-server": "./dist/components.server.js",
+      "default": "./dist/components.client.js"
+    },
+    "#server": {
+      "react-server": "./dist/server-helpers.server.js",
+      "edge-light": "./dist/server-helpers.server.js",
+      "worker": "./dist/server-helpers.server.js",
```

Comment:
I understand this correctly, we are specifying the source file for a specific module (e.g `#server`) based on the runtime it will run on ?

Reply:
Exactly :)
We conditionally import the implementation and we re-export it from the same path, `@clerk/nextjs`